polynote-runtime/src/main/java/polynote/runtime/KernelRuntime.java

```diff
@@ -6,14 +6,29 @@
     public interface Display extends Serializable {
         void content(String contentType, String content);
 
+        // TODO(jindig): I removed write() and a few other methods from here. Should we keep them for backwards-compatibility?
```

Comment:
@jeremyrsmith Any objections to this change? The only usage of the methods I removed that I am aware of was the PythonInterpreter's `sys.stdout = kernel.display`, which is handled in this PR. So it feels safe to remove these methods but this is a "public" API after all. 

WDYT?